packages/nvidia_nat_openpipe_art/src/nat/plugins/openpipe/trainer_adapter.py

```
@@ -0,0 +1,334 @@
+
+        if not health:
+            raise ConnectionError("Failed to connect to ART backend.")
+
+        logger.info("Successfully registered with ART backend.")
+
+    async def is_healthy(self) -> bool:
+        try:
+            async with httpx.AsyncClient() as c:
+                await c.get(f"http://{self.adapter_config.backend.ip}:8000/v1/models",
+                            headers={"Authorization": f"Bearer {self.adapter_config.backend.api_key}"})
+            return True
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)